src/ApparentHorizons/SpherepackIterator.hpp

```diff
@@ -0,0 +1,116 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+///
+/// \file
+/// Defines class SpherepackIterator.
+
+#pragma once
+
+#include <ostream>
+#include <vector>
+
+#include "ErrorHandling/Assert.hpp"
+
+/// \ingroup Spectral
+/// \brief
+/// Iterates over spectral coefficients stored in SPHEREPACK format.
+///
+/// \details
+/// The internal SPHEREPACK ordering is not intuitive, so
+/// SpherepackIterator exists for the purpose of iterating over the
+/// array and determining which specific values of l and m correspond
+/// to each iteration.
```

Comment:
`...purpose of iterating over the SPHEREPACK array and determining the (l,m) of each entry in the array.`